test/unit/rules/lint-img-alt-attributes-test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'img-alt-attributes',
+
+  config: true,
+
+  good: [
+    '<img alt="hullo">',
+    '<img src="hello.jpg" alt="">'
```

Comment:
I know we've had some back and forth on this but I don't think having an empty alt tag is any different than not having one at all.

While we're creating rules for `img` tags, should we enforce an `src` attribute? The spec says it's required. https://www.w3.org/TR/html-markup/img.html